lib/logstash/pluginmanager/install.rb

```diff
@@ -1,82 +1,128 @@
 require 'clamp'
 require 'logstash/namespace'
 require 'logstash/environment'
-require 'logstash/pluginmanager'
 require 'logstash/pluginmanager/util'
-require 'rubygems/dependency_installer'
-require 'rubygems/uninstaller'
 require 'jar-dependencies'
 require 'jar_install_post_install_hook'
 require 'file-dependencies/gem'
 
+require "logstash/gemfile"
+require "logstash/bundler"
+
 class LogStash::PluginManager::Install < Clamp::Command
+  parameter "[PLUGIN] ...", "plugin name(s) or file"
+  option "--version", "VERSION", "version of the plugin to install"
+  option "--force", :flag, "force install without verifying plugin validity"
```

Comment:
let's name this "--[no-]verify" and set `:default => true` (to verify by default). `--force` over time will lose meaning and may confuse.